docs/api/ref/DesktopAgent.md

```diff
@@ -341,6 +342,20 @@ const appIntent = await fdc3.findIntent("StartChat", context);
 await fdc3.raiseIntent("StartChat", context, appIntent.apps[0].name);
 ```
 
+### `raiseContext`
+
+```ts
+raiseContext(context: Context): Promise<IntentResolution>;
+```
+Raises a context to the desktop agent to resolve. Raise context deals with the case where a context can have multiple associated intents. Similar to raiseIntent without a target, it provides the opportunity for Intent and target selection, which can result in a call to raiseIntent with a target under the hood to provide Intent resolution.
```

Comment:
This description doesn't make sense to me, mostly I think because it is not clear what "raising a context" means if you don't understand that under the covers the desktop is looking up intent(s) for the context and asking the user to pick one.

I think it is good if FDC3 (at this stage) only have one "thing" that can be "raised", but there are different ways of doing so, i.e. based on name, and based on context.